server/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "ucsd-cse-210-project-server",
+  "version": "1.0.0",
+  "description": "UCSD CSE 210 Project Server Backend",
+  "main": "cd src && server.js",
+  "scripts": {
+    "start": "node ./src/server.js",
+    "dev": "nodemon ./src/server.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/rohithaug/UCSD-CSE-210-Team-12.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/rohithaug/UCSD-CSE-210-Team-12/issues"
+  },
+  "homepage": "https://github.com/rohithaug/UCSD-CSE-210-Team-12#readme",
+  "dependencies": {
+    "axios": "^1.6.2",
+    "cors": "^2.8.5",
+    "dotenv": "^16.3.1",
+    "express": "^4.18.2",
+    "helmet": "^7.1.0",
+    "http-status": "^1.7.3",
+    "method-override": "^3.0.0"
```

Comment:
Higher versions of these dependencies are available. Do you think we should upgrade these instead?

Reply:
Yeah, we could do that - these are the version I used for the previous project, not sure if updating will have any breaking changes - feel free to update and check, if it works we can update.

Reply:
I updated `axios` and `dotenv` packages. I can confirm that the project starts on port `4000` in development mode as expected. Do you want me to commit this, push, and the approve the merge?

Reply:
Yeah sure